onnx/utils.py

```diff
@@ -224,8 +224,11 @@ def extract_model(
     if not output_names:
         raise ValueError("Output tensor names shall not be empty!")
 
-    onnx.checker.check_model(input_path)
+    if check_model:
+        onnx.checker.check_model(input_path)
     model = onnx.load(input_path)
```

Comment:
Is the external tensors required?

Reply:
That is a good question. There is some room for improvement, but we might need the tensors for the output (which is also specified as just a path). It's a slippery slope to redesigning the entire API (depending on what users want). Keeping it simple here